core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -212,10 +212,7 @@ export class DotContainerPropertiesComponent implements OnInit, AfterViewInit {
                     structureId: new FormControl(state.contentType.variable ?? '', [
                         Validators.required
                     ]),
-                    code: new FormControl(state?.code || '', [
-                        Validators.required,
-                        Validators.minLength(2)
-                    ])
+                    code: new FormControl(state?.code || '')
```

Comment:
Can we test this to make sure no one marks it as `required`?

Reply:
Noted